aerolyzer/retrieve_image_data.py

```diff
@@ -36,28 +38,31 @@ def get_all_exif(self, path):
     '''
     def get_exif(self, path):
         tags = {};
-        allTags = get_all_exif(path)
+        allTags = self._get_all_exif(path)
         for key, value in allTags.iteritems():
             if key in self.data['selectTags']:
                 if type(value) == string:
                     tags[key.lower()] = value.lower()
                 else:
                     tags[key.lower()] = value
-            else:
         return tags
 
     '''
     Purpose:        The purpose of this function is to retrieve the RGB values from an
                     image.
     Inputs:         string path
     Outputs:        None
-    Returns:        array of RGB values
+    Returns:        list of lists of lists containing RGB values
```

Comment:
return a tuple of list instead of list of lists, reason being a tuple is an immutable object. We don't want persons fiddling with this order.